package.json

```diff
@@ -565,6 +565,12 @@
           "description": "Flags to pass to `go test`. If null, then buildFlags will be used.",
           "scope": "resource"
         },
+        "go.saveTestFileOnRunOrDebug": {
```

Comment:
`go test` can only work on files on disk. Therefore, it is reasonable to say that the current file needs to be saved for any test in the current file to be run.

I don't see any reason why one would want to disable this setting.
Unless we have a scenario for it, I would say we don't need this setting.

Reply:
👍